src/modules/blockchainApplication/connection/components/RequestSummary/index.js

```diff
@@ -41,32 +45,37 @@ const RequestSummary = ({ nextStep }) => {
   const [senderAccount, setSenderAccount] = useState(null);
   const { session } = useSession();
   const { moduleCommandSchemas } = useCommandSchema();
+  const metaData = useBlockchainApplicationMeta();
   useDeprecatedAccount(senderAccount);
   useSchemas();
 
+  const sendingChainID = request?.chainId.replace('lisk:', '');
+  const tokenData = useAppsMetaTokens({ config: { params: { chainID: sendingChainID } } });
+
   const approveHandler = () => {
     const moduleCommand = joinModuleAndCommand(transaction);
     const transactionJSON = toTransactionJSON(transaction, moduleCommandSchemas[moduleCommand]);
-    // @todo remove the hardcoded normalized
-    // values according to #4824
+    const { recipientChainID } = request?.request?.params ?? {};
+    const sendingChain = metaData.data.data.find((item) => item.chainID === sendingChainID);
```

Comment:
I would suggest that this is optional chained since during the state transition of `metaData` it could be undefined at first.

```suggestion
    const sendingChain = metaData?.data?.data?.find(({ chainID }) => chainID === sendingChainID);
```